addons/rearm/functions/fnc_addVehicleMagazinesToSupply.sqf

```diff
@@ -29,16 +29,17 @@ if (_vehicle isEqualType objNull) then {_vehicle = typeOf _vehicle};
 if (_vehicle == "") exitWith {
     ERROR_1("VehicleType [%1] is empty in ace_rearm_fnc_addVehicleMagazinesToSupply",_string);
 };
+private _turrets = [_vehicle] call FUNC(getAllRearmTurrets);
```

Comment:
it looks strange...
> -Hey man, you can do it shorter
-I know but I want function
-You can do it faster
-I know but I want function
-You can do it even shorter and faster
-I know but I WANT FUNCTION

I don't know what to say else. I just disagree.